server/testsupport/inPrisonEducationTestDataBuilder.ts

```diff
@@ -46,4 +46,24 @@ const aValidWoodWorkingInPrisonEducation = (): InPrisonEducation => {
   }
 }
 
-export { aValidEnglishInPrisonEducation, aValidMathsInPrisonEducation, aValidWoodWorkingInPrisonEducation }
+const aValidEnglishInPrisonEducationWithinLast12Months = (): InPrisonEducation => {
+  return {
+    prisonId: 'MDI',
+    prisonName: 'Moorland (HMP & YOI)',
+    courseName: 'GCSE English',
+    courseCode: '008ENGL06',
+    courseStartDate: moment('2023-10-01').toDate(),
+    courseCompleted: true,
+    courseCompletionDate: moment('2024-02-26').toDate(),
```

Comment:
Similar to my comment on the unit test, after 26th Feb 2025, this will no longer be "within the last 12 months"
Not sure how to address this yet, so lets leave it for now and we can give it some thought 🤔